src/app/modules/spcp/spcp.util.ts

```diff
@@ -53,6 +53,8 @@ const isArtifactValid = function (
   )
 }
 
+export type RedirectTarget = `${string},${boolean},${string}`
```

Comment:
this may cause the build to fail to compile, suggest
```ts
export type RedirectTarget = `${string},${boolean},${string}` | `${string},${boolean}` 
```
since there are 2 variants